shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -119,52 +119,52 @@
     public static final String UPDATE_STREAM = "controller.stream.updated";                         // Counter and Per-stream Counter
-    public static final String UPDATE_STREAM_LATENCY = "controller.stream.updated_latency_ms";      // Histogram
+    public static final String UPDATE_STREAM_LATENCY = "stream.updated_latency_ms";      // Histogram
     public static final String UPDATE_STREAM_FAILED = "controller.stream.update_failed";            // Counter and Per-stream Counter
     public static final String TRUNCATE_STREAM = "controller.stream.truncated";                     // Counter and Per-stream Counter
-    public static final String TRUNCATE_STREAM_LATENCY = "controller.stream.truncated_latency_ms";  // Histogram
+    public static final String TRUNCATE_STREAM_LATENCY = "stream.truncated_latency_ms";  // Histogram
     public static final String TRUNCATE_STREAM_FAILED = "controller.stream.truncate_failed";        // Counter and Per-stream Counter
 
     // Transaction request Operations
     public static final String CREATE_TRANSACTION = "controller.transactions.created";                      // Counter and Per-stream Counter
-    public static final String CREATE_TRANSACTION_LATENCY = "controller.transactions.created_latency_ms";   // Histogram
+    public static final String CREATE_TRANSACTION_LATENCY = "transactions.created_latency_ms";   // Histogram
     public static final String CREATE_TRANSACTION_FAILED = "controller.transactions.create_failed";         // Counter and Per-stream Counter
     public static final String COMMIT_TRANSACTION = "controller.transactions.committed";                    // Counter and Per-stream Counter
-    public static final String COMMIT_TRANSACTION_LATENCY = "controller.transactions.committed_latency_ms"; // Histogram
+    public static final String COMMIT_TRANSACTION_LATENCY = "transactions.committed_latency_ms"; // Histogram
     public static final String COMMIT_TRANSACTION_FAILED = "controller.transactions.commit_failed";         // Counter, Per-stream Counter, Per-transaction Counter
     public static final String ABORT_TRANSACTION = "controller.transactions.aborted";                       // Counter and Per-stream Counter
-    public static final String ABORT_TRANSACTION_LATENCY = "controller.transactions.aborted_latency_ms";    // Histogram
+    public static final String ABORT_TRANSACTION_LATENCY = "transactions.aborted_latency_ms";    // Histogram
     public static final String ABORT_TRANSACTION_FAILED = "controller.transactions.abort_failed";           // Counter, Per-stream Counter, Per-transaction Counter
     public static final String OPEN_TRANSACTIONS = "controller.transactions.opened";                        // Per-stream Gauge
     public static final String TIMEDOUT_TRANSACTIONS = "controller.transactions.timedout";                  // Per-stream Counter
```

Comment:
same comment as @RaulGracia about removing `controller.` prefix and segmentstore prefix from all names.

Reply:
@shiveshr only those metrics belong to a StatsLogger may have duplication in names, because the loggerName and metric name may both contain things such as "controller" or "segmentstore".
The dynamic metrics don't have statsLogger prefix, so it is essential to have "controller"/"segmentstore" inside names to indicate the origin of the metrics.